nzedb/Movie.php

```diff
@@ -551,7 +556,7 @@ private function checkTraktValue($value)
 			}
 			$value = $temp;
 		}
-		return (isset($value) && !empty($value) ? $value : '');
+		return ($value !== null && !empty($value) ? $value : '');
```

Comment:
We don't have a guarantee here that $value is set. Either revert the change or make sure $value defaults to null in the signature.